### PR TITLE
Blog: PyTorch TMVA Interface release

### PR DESCRIPTION
**Pull request by AnirudhDagar**
Add blog for the newest addition to PyMVA supported ML frameworks in the form of PyTorch with `ROOT v6-24-00 `.
@lmoneta Let me know your thoughts and feedback. :)

### CODE REVIEW

**Comment by eguiraud** [comment]
Hi @AnirudhDagar , thank you for the contribution! Please check previous blog PRs (e.g. #524) for how to include your author information so that it's properly displayed when the blog post is published. Also the first sentence should be a brief abstract/advert for the post as it is featured on the website home page. I'll leave it to Lorenzo to review the actual content :)

File: blog/_posts/2021-03-19-tmva-pytorch.md
Comment (eguiraud) [inline blog/_posts/2021-03-19-tmva-pytorch.md:7]:
```suggestion
What if we combine [PyTorch][PyTorch] and [TMVA][TMVA]? Ever wondered how ROOT utilizes powerful external MVA libraries making them easily accessible with a direct integration into the TMVA workflow? These interfaces between TMVA and Python frameworks are powered by the PyMVA backbone. All PyMVA methods provide the same plug-and-play mechanisms as TMVA.
```

File: blog/_posts/2021-03-19-tmva-pytorch.md
Comment (eguiraud) [inline blog/_posts/2021-03-19-tmva-pytorch.md:18]:
```suggestion
TMVA already has a PyKeras interface which we all love, especially with Keras’ simple high-level tensorflow API. If your work involves some elementary experiments, Keras maybe the goto framework due to its plug and play spirit.
```

File: blog/_posts/2021-03-19-tmva-pytorch.md
Comment (eguiraud) [inline blog/_posts/2021-03-19-tmva-pytorch.md:20]:
```suggestion
But things get interesting when one requires low level control and flexibility. That’s when the argument for Keras starts losing water. PyTorch on the other hand is amazing in terms of control, flexibility and raw power that it can provide to the user. Its lower-level approach is better suited for the more mathematically-inclined users.
```

File: blog/_posts/2021-03-19-tmva-pytorch.md
Comment (eguiraud) [inline blog/_posts/2021-03-19-tmva-pytorch.md:30]:
```suggestion
* Performance: PyTorch is extremely fast due to its highly optimized C++ backend.
```

File: blog/_posts/2021-03-19-tmva-pytorch.md
Comment (eguiraud) [inline blog/_posts/2021-03-19-tmva-pytorch.md:33]: "Neural Net" was capitalized before. I would never capitalize it, but anyway we should be consistent

**Review by eguiraud — CHANGES_REQUESTED** [review]
minor text fixes

**Review by lmoneta — APPROVED** [review]
Looks good to me !
Thank you Anirudh for providing this blog

File: blog/_posts/2021-03-19-tmva-pytorch.md
Comment (AnirudhDagar) [reply blog/_posts/2021-03-19-tmva-pytorch.md]: I capitalized it because it was at the start of the sentence. Anyways replaced it with "model", should be good to go.

**Comment by eguiraud** [comment]
Thanks @AnirudhDagar and @lmoneta ! The last issue: the title says the PyTorch-TMVA interface is "out", but v6.24 is not released yet. Do you want to hold off the publication of the post until v6.24 is released (in a few weeks) or would you rather have the blog post out sooner than the feature?

**Comment by AnirudhDagar** [comment]
I think @lmoneta can take a call on this. I'll be happy to wait for the release or tweak a few things to avoid explicitly mentioning the release if we are to merge now.

**Comment by lmoneta** [comment]
I think is better to publish the blog now before the release. 
We can change maybe the title to something else like : 
**New PyTorch interface in TMVA**  

and in the text later we write:

*with the coming release of ROOT 6-24 we are excited to...*

**Comment by AnirudhDagar** [comment]
@eguiraud @lmoneta I've made the changes suggested so that we can merge it before the v6.24 release.

**Comment by lmoneta** [comment]
Thank you Anirudh!
 @eguiraud if it is also fine with you, please merge it

**Comment by eguiraud** [comment]
Thanks @AnirudhDagar , merging!

**Pull request merged**
